etc/production.bash

```diff
@@ -138,6 +138,14 @@ tar xf libs.tar
 cp ${etc}/emotiq.bash.${kind} ${production_dir}/emotiq.bash
 cd ${lib}
 rm *
-# < /test >
 
-# cp ${???}/*.so* ${production_dir}
+cd ${production_dir}
+case ${kind} in
+    linux) 
+	tar cfj ${kind}.emotiq.bz2 em* *.so*
```

Comment:
This is an unusual naming pattern for software archives.  

Please name the resulting archive
```
emotiq-${version}-${kind}.tar.bz2
```
to indicate that this is a tar archive.  

For `version` I would either use a timestamp of sufficient precision (minutes) to be unique, or read the contents of a file in git, or interogate the branch name.  You may want to ask Vlad/Eugene for guidance on how we wish to version our releases.

`kind` would be better named `arch`.

Reply:
OK to both.  I will change this stuff.